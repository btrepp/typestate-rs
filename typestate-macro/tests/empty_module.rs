use typestate::typestate;

#[typestate]
mod m {}

fn main() {}