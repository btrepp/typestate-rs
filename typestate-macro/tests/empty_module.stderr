error: Missing `#[automata]` struct.
 --> $DIR/empty_module.rs:3:1
  |
3 | #[typestate]
  | ^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
